src/main/resources/view/MainWindow.fxml

```diff
@@ -53,12 +43,16 @@
           <tabs>
             <Tab fx:id="patientTab" text="Patients">
               <content>
-                <StackPane fx:id="patientListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
+                <StackPane fx:id="patientListPanelPlaceholder" VBox.vgrow="ALWAYS" />
+
               </content>
             </Tab>
             <Tab fx:id="roomTab" text="Rooms">
               <content>
-                <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="180.0" prefWidth="200.0" />
+                <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="180.0" prefWidth="200.0">
+                           <children>
```

Comment:
Do you think we should keep the indentation consistent? For *.fxml files, maybe we can use 2 or 4 spaces (not sure what the original is).

Reply:
yup, i will change the indentation. good catch